### PR TITLE
refactor setting up cluster using kubeadm

### LINKED ISSUE #7105: Refacor parts of kubeadm docs
**This is a...** 

- [ ] Feature Request
- [ ] Bug Report
- [ x ] Others

**Problem:**
Recently , many issues about set up cluster using kubeadm, i think we should make docs more clear.

**Proposed Solution:**
The root reason are most people do not know how to use it, and  `cgroup-driver` diff between `kubelet config` and  `docker`, we should make it more clear.

**Page to Update:**
https://kubernetes.io/docs/setup/independent/install-kubeadm/
https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/
https://kubernetes.io/docs/setup/independent/troubleshooting-kubeadm/
https://kubernetes.io/docs/setup/independent/high-availability/

### PR DESCRIPTION
**Pull request by stewart-yu**
Many issues about error when using kubeadm set up cluster, so it need to make docs more clear.
fix #7105 

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7104)

### CODE REVIEW

**Comment by stewart-yu** [comment]
cc @luxas @xiangpengzhao PTAL, thanks

**Comment by xiangpengzhao** [comment]
@stewart-yu thanks for the update. Please resolve the conflict before we start the review :)

**Comment by stewart-yu** [comment]
Done, PTAL @xiangpengzhao thanks

**Comment by xiangpengzhao** [comment]
@stewart-yu The status shows that `This branch cannot be rebased due to conflicts`. Please double-check it :)

And the commit `Merge branch 'master' into patch-2` was generated by `git pull`, right? `git rebase upstream/master` seems a better way to rebase for not generating such a commit.

**Comment by stewart-yu** [comment]
Done, PTAL @xiangpengzhao

File: docs/setup/independent/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -58,11 +58,18 @@ This may be caused by a number of problems. The most common are:
 
   There are two common ways to fix the cgroup driver problem:
   
- 1. Install docker again following instructions
+ 1. Install docker again following intstructions
```
Comment (xiangpengzhao) [inline docs/setup/independent/troubleshooting-kubeadm.md]: typo

Comment (stewart-yu) [reply docs/setup/independent/troubleshooting-kubeadm.md]: Done, thanks @xiangpengzhao

File: docs/setup/independent/install-kubeadm.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
 {% endcapture %}
 
 {% capture prerequisites %}
+## Pre-Requisites
```
Comment (jamiehannaford) [inline docs/setup/independent/install-kubeadm.md]: The `{% capture prerequisites %}` seems to be a global rule, so if you're unhappy about how that's formatted, we should probably change it there

See https://github.com/kubernetes/website/blob/52a1b9f7ab84812faad4fd3ad7ad243cb6c3e65d/_includes/templates/task.md#L18

File: docs/setup/independent/install-kubeadm.md [hunk 3]
```diff
@@ -23,32 +24,32 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
 * 2 GB or more of RAM per machine (any less will leave little room for your apps)
 * 2 CPUs or more 
 * Full network connectivity between all machines in the cluster (public or private network is fine)
-* Unique hostname, MAC address, and product_uuid for every node
-* Certain ports are open on your machines. See the section below for more details
+* Unique hostname, MAC address, and product_uuid for every node. See the section below for more details.
```
Comment (jamiehannaford) [inline docs/setup/independent/install-kubeadm.md]: can we make "section below" an anchor link?

File: docs/setup/independent/install-kubeadm.md [hunk 4]
```diff
@@ -23,32 +24,32 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
...
+* Certain ports are open on your machines. See the section below for more details.
```
Comment (jamiehannaford) [inline docs/setup/independent/install-kubeadm.md]: same link, a link is better

File: docs/setup/independent/install-kubeadm.md [hunk 5]
```diff
@@ -23,32 +24,32 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
...
 * Swap disabled. You **MUST** disable swap in order for the kubelet to work properly. 
 
 {% endcapture %}
 
 {% capture steps %}
 
-## Verify the MAC address and product_uuid are unique for every node
+#### Verify the MAC address and product_uuid are unique for every node
```
Comment (jamiehannaford) [inline docs/setup/independent/install-kubeadm.md]: why are making these h4?

File: docs/setup/independent/install-kubeadm.md [hunk 6]
```diff
@@ -23,32 +24,32 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
...
+#### Check network adapters
 
 If you have more than one network adapter, and your Kubernetes components are not reachable on the default
 route, we recommend you add IP route(s) so Kubernetes cluster addresses go via the appropriate adapter.
 
-## Check required ports
+##### Check required ports
 
-### Master node(s)
+###### Master node(s)
```
Comment (jamiehannaford) [inline docs/setup/independent/install-kubeadm.md]: do we use h6s anywhere else on the site? maybe this just needs to be bold?

File: docs/setup/independent/install-kubeadm.md (hunk 3 above)
Comment (stewart-yu) [reply docs/setup/independent/install-kubeadm.md]: done, thanks

File: docs/setup/independent/install-kubeadm.md (hunk 2 above)
Comment (stewart-yu) [reply docs/setup/independent/install-kubeadm.md]: revert modify, thanks

**Comment by heckj** [comment]
@jamiehannaford The content has been updated - would you look it over for a technical accuracy review please?

@xiangpengzhao are you satisfied with the updates based on your feedback?

/assign

**Comment by xiangpengzhao** [comment]
LGTM. Defer to @jamiehannaford for final confirmation :)

**Pull request merged**